### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by SpicyWasab**
Hi !

I just found this app in today's Gnome Software recommendations a few dozen minutes ago. it looked pretty and I think I might use it to replace the default Weather app.

I noticed there isn't any French translation yet, so here it is. However I'm unsure about some of the translations I put into it (may be the wrong context or too long), so I'll see if/when it gets implemented if I should correct some strings.
I'm also not that familiar with .po files, so I might have done something wrong in the preamble of the file. (?) I used the es.po file to have some more context about the allowed length for each string.

Thanks for the beautiful UI c:
(and looking forward for that Dynamic Gradient thing :P)

### CODE REVIEW

**Comment by amit9838** [comment]
Hi @SpicyWasab  Actually .pot file or es.po are not updated with current version, so this translation won't have any effect on the app. Pls wait unitll we update .pot file.

File: po/fr.po [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+msgstr "Utiliser des dégradés de couleurs dynamiques basés sur les conditions météo actuelles pour le fond."
+
+#: data/io.github.amit9838.weather.appdata.xml.in:13
+#: data/io.github.amit9838.weather.appdata.xml.in:74
+msgid "See forcasts in beautiful card view."
+msgstr "Afficher les prévisions dans des cartes esthétiques."
+
+#: data/io.github.amit9838.weather.appdata.xml.in:14
+msgid "Use Personal API Key"
+msgstr "Utiliser une clé d'API personnelle"
```
Comment (amit9838) [inline po/fr.po:50]: for example this code block. Weather dosen't use api key anymore.

**Comment by SpicyWasab** [comment]
Oh, okay.
I'll come back when the new version is out then c:

(Yeah I noticed this API key thing and I was pretty confused x))

**Comment by amit9838** [comment]
Hi @SpicyWasab  I've updated the .pot file with the latest version of the app, hopefully you'll be able to translate.

**Pull request closed without merging**